### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by RishavRanjan123** [community/community#59998] [GitHub Education]
### Hi everyone,

I am a user of github student developer pack since 1 year. And, now the services has stopped for me to re-verify my credentials to prove that i'm still a part of my college. I uploaded my id card pic and it's getting rejected for two reasons.
First, I've completed my profile already, still it's saying to complete it. Second, it says that my location doesn't match with college location, but i am on my vacations now, how is it supposed to not to be a case?

**Comment by samus-aran** [comment]
Hi @goldithehero 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 I suspect this would be helpful.

It would be preferable that a student is on campus when applying and thus able to upload a document. Understandable you are on vacation and recommend you apply when you return closer to campus.
